cli/src/test/scala-2.12/coursier/cli/CliFetchIntegrationTest.scala

```diff
@@ -414,15 +414,15 @@ class CliFetchIntegrationTest extends FlatSpec with CliTestLib {
    * Result:
    * |└─ org.apache.commons:commons-compress:1.5
    */
-  "local dep url" should "have coursier-fetch-test.jar" in withFile() {
+  "local file dep url" should "have coursier-fetch-test.jar and cached for second run" in withFile() {
     (jsonFile, _) => {
       withFile("tada", "coursier-fetch-test", ".jar") {
         (testFile, _) => {
           val path = testFile.getAbsolutePath
           val encodedUrl = encode("file://" + path, "UTF-8")
 
 
-          val commonOpt = CommonOptions(jsonOutputFile = jsonFile.getPath)
+          val commonOpt = CommonOptions(jsonOutputFile = jsonFile.getPath, cacheFileArtifacts = true)
```

Comment:
Given this the first run should put the artifact in cache already, right? so there's no need for second run.

Reply:
The second part of this test checks to see that what was fetched was actually from the cache and not the local file.

Reply:
Ah I see. sounds good!